trillian_log_api.proto

```diff
@@ -138,9 +138,30 @@ service TrillianLog {
     }
 }
 
+// Charge describes the user(s) associated with the request whose quota should
```

Comment:
Maybe `ChargeTo` ? ("Charge" makes me think it describes the amount rather than the payee)

Reply:
Good call, that reads a lot better - thanks!